src/shutdown/shutdown.c

```diff
@@ -463,8 +476,9 @@ int main(int argc, char *argv[]) {
                                 log_error_errno(r, "Failed to detach DM devices: %m");
                 }
 
-                if (!need_umount && !need_swapoff && !need_loop_detach && !need_dm_detach) {
-                        log_info("All filesystems, swaps, loop devices and DM devices detached.");
+                if (!need_umount && !need_swapoff && !need_loop_detach && !need_dm_detach
+                            && !need_md_detach) {
+                        log_info("All filesystems, swaps, loop devices, DM arrays and DM devices detached.");
```

Comment:
"DM arrays" ??  Did you mean "MD arrays"??  But you call them "MD devices" in other messages.
"Md devices" is best in my opinion.

Reply:
fixed